### DISCUSSION TITLE
mini-files: Nothing happens when pressing `=` after deleting a file

### DISCUSSION THREAD

**Discussion by idelice** [nvim-mini/mini.nvim#845] [General]
I'm using LazyVim, and I'm trying to delete one or more files.

I see the files deleted in the mini.files window, and then I press `=` but nothing happens.

I tried quitting it and hoping it would synchronize the changes but nope.

What am I doing wrong here? I just want to edit/delete buffers?

**Comment by echasnovski** [comment]
> What am I doing wrong here? I just want to edit/delete buffers?

Sorry, there is not enough reproducible steps to answer anything other than "*Probably* *something* is done wrong".

Pressing `=` when explorer is open should synchronize changes with file system after confirmation dialog.

**Reply by idelice** [reply]
Never mind.. My keyboard layer was messing with me... 😆
